src/command_modules/azure-cli-rdbms/setup.py

```diff
@@ -12,7 +12,7 @@
     logger.warn("Wheel is not available, disabling bdist_wheel hook")
     cmdclass = {}
 
-VERSION = "0.3.5"
+VERSION = "0.3.6"
```

Comment:
JFYI, moving forward, updating `setup.py` is no longer needed. In history.rst file, you just add `* Improve help message and command parameters` on top of the existing version, and new header with the version is not necessary either

Reply:
Thanks! good to know.